nuget/lib/dependabot/nuget/metadata_finder.rb

```diff
@@ -12,16 +12,148 @@ module Nuget
     class MetadataFinder < Dependabot::MetadataFinders::Base
       extend T::Sig
 
+      sig do
```

Comment:
There is no functional change to this file, it is simply being restored to its state from 2 weeks ago.